### PR TITLE
Form builder

### PR DESCRIPTION
**Pull request by Codemax999**

### CODE REVIEW

File: Fenner_William-DFBT/js/index.js [hunk 1]
```diff
@@ -42,6 +41,7 @@ const form_builder = (attach, ...form_elements) => {
 
     }
 
+    //Close the form element
     form_string += `</form>`;
 
     if(attach == ""){
```
Comment (Codemax999) [inline Fenner_William-DFBT/js/index.js]: Project should return string only, instead of appending to DOM

**Comment by Codemax999** [comment]
Include a .gitignore file with `/node_modules/` in it to remove node_modules from GitHub. Typically node_modules is not something added to version control.

File: Fenner_William-DFBT/js/index.js [hunk 2]
```diff
@@ -1,56 +1,59 @@
...
+    form_string += `</form>`;
+
     if(attach == ""){
         //If no particular element is specified to add this to...
-        document.body.appendChild(form);
+        document.body.innerHTML += form_string;
         //console.log(form.innerHTML);
     }
     else{
         //document.getElementsByName(attach) += form;
```
Comment (Codemax999) [inline Fenner_William-DFBT/js/index.js]: Remove commented out code.

**Comment by wsf235** [comment]
Fixed a couple spots and removed some unneeded folders

**Comment by Codemax999** [comment]
Awesome. LGTM

**Pull request merged**